### PR TITLE
Put the authentication credentials into the specified namespace

### LINKED ISSUE #120: Put the authentication credential secret in a specified namespace is easier to use.
**What would you like to be added**:

Put the authentication credential secret in the specified namespace, and read it from here when using it.
This is easier to use and easier to maintain.

**Why is this needed**:

I see that HUAWEI CLOUD authentication credential secret must be created in the namespace of the ELB service, which is not very convenient to use. 
If ELB services are created in multiple namespaces, we need to create the same secret in each namespace, which is inconvenient.

### PR DESCRIPTION
**Pull request by chengxiangdong**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

I see that the HUAWEI CLOUD authentication credential secret must be created into the namespace of the ELB service, which is not very convenient to use. 
Putting the authentication credential secret in the specified namespace is easier to use.

**Which issue(s) this PR fixes**:

Fixes #120

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by chengxiangdong** [comment]
@RainbowMango 
I refactored the code, please review

File: hack/pre-run-e2e.sh [hunk 1]
```diff
@@ -15,13 +15,8 @@
 # limitations under the License.
 
 set -o errexit
-set -o pipefail
-
-if [[ "${HC_ACCESS_KEY}" = "" ]] || [[ "${HC_SECRET_KEY}" = "" ]]; then
-  echo "HC_ACCESS_KEY and HC_SECRET_KEY can not be empty, please set it"
```
Comment (niuzhenguo) [inline hack/pre-run-e2e.sh:21]: why remove this?

Comment (chengxiangdong) [reply hack/pre-run-e2e.sh:21]: the secret has been created in the specified namespace, so we do not need to create it every time we run the E2E tests.

Comment (niuzhenguo) [reply hack/pre-run-e2e.sh:21]: ok

**Pull request merged**